corehq/apps/app_manager/templates/app_manager/v1/form_view_base.html

```diff
@@ -118,6 +118,7 @@
             return result;
         }
 
+        var form_requires = ko.observable({{ form.requires|JSON }});
```

Comment:
What's this guy for?  Looks like elsewhere you've been using `initial_page_data` for it.